projects/packages/forms/src/blocks/contact-form/components/jetpack-field-option.js

```diff
@@ -41,21 +48,46 @@ export const JetpackFieldOptionEdit = props => {
 		removeBlock( clientId );
 	};
 
+	const onFocus = () => {
+		// TODO: move cursor to end
```

Comment:
When you delete an option, the previous one gets the focus, but the caret stays at the start. I've tried to move it at the end and select the entire text but couldn't eventually make it work, though both approaches are documented online numerous times. I don't know what's blocking here. Even `requestAnimationFrame` and `setTimeout` calls didn't work.